radis/test/lbl/test_factory.py

```diff
@@ -727,62 +730,66 @@ def test_vaex_and_pandas_spectrum():
 
-        s, factory_s = calc_spectrum(
-            1800,
-            1820,  # cm-1
-            molecule="CO",
-            isotope="1",
-            pressure=1.01325,  # bar
-            Tgas=700,  # K
-            Tvib=710,
-            Trot=710,
-            mole_fraction=0.1,
-            wstep="auto",
-            path_length=1,  # cm
-            databank="hitemp",  # or 'hitemp', 'geisa', 'exomol'
-            optimization=None,
-            engine="vaex",
-            verbose=3,
-            return_factory=True,
-        )
+    from radis import calc_spectrum
 
-        s.apply_slit(0.5, "nm")  # simulate an experimental slit
+    # Calculating spectrum using vaex
+    s, factory_s = calc_spectrum(
+        1800,
+        1820,  # cm-1
+        molecule="CO",
+        isotope="1",
+        pressure=1.01325,  # bar
+        Tgas=700,  # K
+        Tvib=710,
+        Trot=710,
+        mole_fraction=0.1,
+        wstep="auto",
+        path_length=1,  # cm
+        databank="hitemp",  # or 'hitemp', 'geisa', 'exomol'
+        optimization=None,
+        engine="vaex",
+        verbose=3,
+        return_factory=True,
+    )
 
-        s1, factory_s1 = calc_spectrum(
-            1800,
-            1820,  # cm-1
-            molecule="CO",
-            isotope="1",
-            pressure=1.01325,  # bar
-            Tgas=700,  # K
-            Tvib=710,
-            Trot=710,
-            mole_fraction=0.1,
-            wstep="auto",
-            path_length=1,  # cm
-            databank="hitemp",  # or 'hitemp', 'geisa', 'exomol'
-            engine="pandas",
-            verbose=3,
-            return_factory=True,
-        )
+    s.apply_slit(0.5, "nm")  # simulate an experimental slit
+
+    # Calculating spectrum using pandas
+    s1, factory_s1 = calc_spectrum(
+        1800,
+        1820,  # cm-1
+        molecule="CO",
+        isotope="1",
+        pressure=1.01325,  # bar
+        Tgas=700,  # K
+        Tvib=710,
+        Trot=710,
+        mole_fraction=0.1,
+        wstep="auto",
+        path_length=1,  # cm
+        databank="hitemp",  # or 'hitemp', 'geisa', 'exomol'
+        engine="pandas",
+        verbose=3,
+        return_factory=True,
+    )
 
-        s1.apply_slit(0.5, "nm")  # simulate an experimental slit
+    s1.apply_slit(0.5, "nm")  # simulate an experimental slit
 
-        import numpy as np
+    import numpy as np
 
-        assert np.allclose(s.get("radiance"), s1.get("radiance"), equal_nan=True)
-        assert np.allclose(s.get("absorbance"), s1.get("absorbance"), equal_nan=True)
-        assert np.allclose(
-            s.get("transmittance"), s1.get("transmittance"), equal_nan=True
-        )
+    # Comparing different quantities
+    assert np.allclose(s.get("radiance"), s1.get("radiance"), equal_nan=True)
+    assert np.allclose(s.get("absorbance"), s1.get("absorbance"), equal_nan=True)
+    assert np.allclose(s.get("transmittance"), s1.get("transmittance"), equal_nan=True)
```

Comment:
no need to apply a slit, and checking only one quantity such as "absorbance" is enough :)